frontend/src/components/TodoListItem.tsx

```diff
@@ -1,20 +1,42 @@
+import { ChangeEvent, useState } from "react";
 import { Todo } from "../models/todo.model";
 import { IconDelete } from "./Icons/IconDelete";
 
 interface TodoListItemProps {
   todo: Todo;
-  deleteTodo: (id: string) => void;
+  onDeleteTodo: (id: string) => void;
+  onChangeTodo: (id: string, check: boolean) => void;
 }
-export default function TodoListItem({ todo, deleteTodo }: TodoListItemProps) {
+export default function TodoListItem({
+  todo,
+  onChangeTodo,
+  onDeleteTodo,
+}: TodoListItemProps) {
+  const [check, setCheck] = useState(!todo.isDone);
+  const valueChangeHandler = (event: ChangeEvent<HTMLInputElement>) => {
+    onChangeTodo(todo.id, event.target.checked);
+    setCheck(!check);
+  };
+
   return (
     <li>
       <div className="list-item flex-container">
-        <p className="list-item-p">{todo.description}</p>
+        <input
+          className="check-box"
+          type="checkbox"
+          onChange={valueChangeHandler}
+          checked={!check}
```

Comment:
Use the `isDone` property of a Todo for `checked`, and also for rendering the description in the appropriate way below